force-app/main/default/lwc/apexRecordsUtils/__tests__/apexRecordsUtils.test.js

```diff
@@ -0,0 +1,28 @@
+import { flatObjectsInArray } from 'c/apexRecordsUtils';
+
+describe('c-apex-records-utils', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('receives the array of objects that it have objects how to child, and it create a property for each child', () => {
```

Comment:
make use of translators or chatgpt to make sure your english is good on code comments